assets/eip-4675/contracts/math/SafeMath.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: CC0-1.0
```

Comment:
Correct me if I'm wrong, but this looks like OpenZeppelin. You cannot change the license of code you do not own.

Reply:
This is solidity v0.8.0... why is OZ SafeMath used anyway???

Reply:
Problem solved. Needs manual merge now.